braindecode/datasets/datasets.py

```diff
@@ -44,6 +47,22 @@ def _fetch_and_unpack_moabb_data(dataset, subject_ids):
     return raws, description
 
 
+def _annotations_from_raw(raw, dataset):
+    # find events from stim channel
+    events = mne.find_events(raw)
+
+    # get annotations from events
+    event_desc = {k: v for v, k in dataset.event_id.items()}
+    annots = mne.annotations_from_events(events, raw.info['sfreq'], event_desc)
+
+    # set trial on and offset given by moabb
```

Comment:
If this function is specific to MOABB datasets, then maybe the function name should also reflect that.

Reply:
Yes, I also agree. I used to have moabb in the name but somehow changed it.